specification/riskiq/Easm/models.tsp

```diff
@@ -270,6 +323,39 @@ model AssetUpdateData {
   transfers?: AssetUpdateTransfers;
 }
 
+@doc("A request body used to export an asset.")
+@Versioning.added(Versions.v2024_03_01_preview)
+model AssetsExportRequestBody {
+  #suppress "@azure-tools/typespec-azure-core/documentation-required" "Documentation will be added next preview version"
```

Comment:
I don't think this "Documentation will be added next preview version" is a valid approach.